### ISSUE TITLE
Mapping Logits Dimensions to Specific Amino Acids

### ISSUE THREAD

**Issue by Cassie818** [allmanbrent/NLP_viral_escape#1]
Hi,

I noticed that in the language_models/ESM_probability_calcs.py script, an amino acid dictionary is defined. However, when applying the softmax function to the logits, it returns a 33-dimensional output corresponding to the 33 tokens defined in ESM2. But in your code, you've only defined the common 20 amino acids. Could you kindly clarify how each of these dimensions is paired with a specific amino acid? Thank you very much.
